pages/glossary/context.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Context
+key: context
+---
+
+The context of a piece of information is any other information that helps a user understand it by clarifying its meaning.
+
+Both the [visual context](#visual-context) and the [programmatically determined context](#programmatically-determined-context) contribute to the context.
+
+**Note**: To be considered as context, content must be available similarly to all users with similar abilities. Thus, for example, content from another [document] is not part of the context as there is no guarantee that all users have access to it.
```

Comment:
Is is possible to simply say that to be considered context, the content must be on the same web page?

Reply:
I'm not sure it's that clear… 

If I'm looking at a website about movies, it's likely that the word "theatre" means "movie theatre" while on a WW2 history site, it would mean "Theatre of war (Europe/Pacific)". In that case, the topic of the website would be context. Or maybe we consider that every single page of the site is about that topic and has enough information to make the difference? Will it always be that clear?

(that note was added after Brian's remark that "a book [user] read 5 years ago should no be context", hence the formulation).